README.md

```diff
@@ -1,159 +1,159 @@
-- [Azure Landing Zones, Enterprise-Scale Reference Implemenation using Terraform](https://github.com/Azure/caf-terraform-landingzones-starter/tree/starter/enterprise_scale/construction_sets/aks/online/aks_secure_baseline)
-
-## Advanced topics
-
-This reference implementation intentionally does not cover more advanced scenarios. For example topics like the following are not addressed:
-
-- Cluster lifecycle management with regard to SDLC and GitOps
-- Workload SDLC integration (including concepts like [Bridge to Kubernetes](https://docs.microsoft.com/visualstudio/containers/bridge-to-kubernetes?view=vs-2019), advanced deployment techniques, etc)
-- Container security
-- Multiple (related or unrelated) workloads owned by the same team
-- Multiple workloads owned by disparate teams (AKS as a shared platform in your organization)
-- Cluster-contained state (PVC, etc)
-- Windows node pools
-- Scale-to-zero node pools and event-based scaling (KEDA)
-- [Terraform](https://docs.microsoft.com/azure/developer/terraform/create-k8s-cluster-with-tf-and-aks)
-- [Bedrock](https://github.com/microsoft/bedrock)
-- [dapr](https://github.com/dapr/dapr)
-
-Keep watching this space, as we build out reference implementation guidance on topics such as these. Further guidance delivered will use this baseline AKS implementation as their starting point. If you would like to contribute or suggest a pattern built on this baseline, [please get in touch](./CONTRIBUTING.md).
-
-## Final thoughts
-
-Kubernetes is a very flexible platform, giving infrastructure and application operators many choices to achieve their business and technology objectives. At points along your journey, you will need to consider when to take dependencies on Azure platform features, OSS solutions, support channels, regulatory compliance, and operational processes. **We encourage this reference implementation to be the place for you to _start_ architectural conversations within your own team; adapting to your specific requirements, and ultimately delivering a solution that delights your customers.**
-
-## Related documentation
-
-- [Azure Kubernetes Service Documentation](https://docs.microsoft.com/azure/aks/)
-- [Microsoft Azure Well-Architected Framework](https://docs.microsoft.com/azure/architecture/framework/)
-- [Microservices architecture on AKS](https://docs.microsoft.com/azure/architecture/reference-architectures/containers/aks-microservices/aks-microservices)
-
-## Contributions
-
-Please see our [contributor guide](./CONTRIBUTING.md).
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact <opencode@microsoft.com> with any additional questions or comments.
-
-With :heart: from Microsoft Patterns & Practices, [Azure Architecture Center](https://aka.ms/architecture).
```

Comment:
Looks like you changed line endings again.  Any reason?  If you have a local configuration thing going on, you can make this edit right in the GitHub UI, that won't cause line ending changes, IIRC.

Reply:
Yes, sorry for that... fixed